src/app/models/login.server.model.ts

```diff
@@ -52,6 +53,20 @@ const LoginSchema = new Schema<ILoginSchema>(
   },
 )
 
+LoginSchema.statics.addLoginFromForm = function (
+  this: ILoginModel,
+  form: IPopulatedForm,
+): Promise<ILoginSchema> {
+  const login = new this({
+    form: form._id,
+    admin: form.admin._id,
+    agency: form.admin.agency._id,
+    authType: form.authType,
+    esrvcId: form.esrvcId,
+  })
+  return login.save()
```

Comment:
nit: do we want to use `this.create` syntax instead of `new`? The former has typings.